test/functional/Java9andUp/src/org/openj9/test/varhandle/StaticFieldVarHandleTests.java

```diff
@@ -2484,4 +2491,226 @@ public void testCommonCallSite() {
 	private long commonVarHandleCallSite(VarHandle vh, long value) {
 		return (long)vh.get();
 	}
+	
+	/**
+	 * Perform all the operations available on a StaticFieldVarHandle referencing a {@link String} field of a
+	 * parent class via the child class.
+	 */
+	@Test
+	public void testReferenceInParentFromChild() {
```

Comment:
Can this be broken into a series of tests?  It will be easier to focus debugging efforts in the future if we can clearly see which operation failed.

Reply:
Divided into a series of tests.